src/content/en/2020/markup.md

```diff
@@ -983,7 +983,7 @@ The two most popular ones stand out because they are almost twice as popular tha
 Attributes like `data-type`, `data-id`, and `data-src` can have multiple generic uses although `data-src` is used a lot with lazy image loading via JavaScript (e.g., Bootstrap 4). [Bootstrap](https://getbootstrap.com/) again explains the presence of `data-toggle`, where it's used as a state styling hook on toggle buttons. The [Slick carousel plugin](https://kenwheeler.github.io/slick/) is the source of `data-slick-index`, whereas `data-element_type` is part of [Elementor's WordPress website builder](https://elementor.com/). Both `data-requiremodule` and `data-requirecontext`, then, are part of [RequireJS](https://requirejs.org/).
 
 {# TODO(authors): Update this interpretation given that the lazy loading stat is in terms of pages, not img elements. #}
-Interestingly, the use of native lazy loading on images is similar to that of `data-src`. 3.86% of pages use the `<img loading="lazy">` attribute. This appears to be growing very fast, as back in February, this number was about [0.8%](https://twitter.com/zcorpan/status/1237016679667970050). It's possible that these are being used together for a [cross-browser solution](https://addyosmani.com/blog/lazy-loading/).
+Interestingly, the use of native lazy loading on images is similar to that of `data-src`. 3.86% of pages use `loading="lazy"` on `img` elements. This appears to be growing very fast, as back in February, this number was about [0.8%](https://twitter.com/zcorpan/status/1237016679667970050). It's possible that these are being used together for a [cross-browser solution](https://addyosmani.com/blog/lazy-loading/).
```

Comment:
@Tiggerito, I couldn’t find any reference for what we looked at in the results spreadsheet. Was this use of `loading` actually limited to `img` elements, or also to `iframe`s? Just to err on the safe side.

Reply:
I added a link in #1858 as also found it difficult to substantiate that figure. I believe this was reused from @Tiggerito 's SEO analysis: https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/edit#gid=2109061092 as it has this 3.86% stat.

That uses this query, which is just for images: https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/07_SEO/pages_markup_by_device_and_img_loading.sql

Reply:
Yes, it's just for images. I separately gathered the same data for iframes.

pages_markup_by_device_and_iframe_loading.sql

Reply:
Thanks for checking!